netmanager/src/config/urls/predict.js

```diff
@@ -4,4 +4,4 @@ const BASE_PREDICT_URL = stripTrailingSlash(
   process.env.REACT_APP_BASE_PREDICT_URL || process.env.REACT_APP_BASE_URL
 );
 
-export const GET_HEATMAP_DATA = `${BASE_PREDICT_URL}/predict/heatmap`;
+export const GET_HEATMAP_DATA = `${BASE_PREDICT_URL}/predict/heatmap?airqloud=kampala`;
```

Comment:
Thanks @FaithDaka , please shed some light on this hardcoded value of `airqloud=kampala`?  Thanks.